Codo-Baggins/covid_comparisons
# Feature/api calls

## What's this PR do?
- This PR fetches coronavirus API data for both the entire United States and the state of Colorado

## Where should the reviewer start?
- The reviewer should first start by looking at the APICalls.tsx file and then navigate over to the `componentDidMount()` function on App.tsx

## How should this be manually tested?
- Clone down the repo
- `cd` into the cloned down directory
- Run `npm start` 
- Navigate to the localhost URL provided in the terminal
- Open up the dev tools and open the `Components` inspection tool 
- Click on any of the data that is displayed onto the page and observe what the raw data looks like in the console

## Any background context you want to provide?
-  Styling still has not been done.  This creates some readability issues for the time being.  

## Issues Resolved 
- Fixes #2 
- Fixes #17
